commit 4c9a583301c9a3683224e5ab32bad1b7271cf403
Author: Peter Michael Green <plugwash@debian.org>
Date:   Sat Sep 4 19:42:56 2021 +0000

    Add expected result for test_text_width in the case where unicode-width is enabled and strip-ansi-escapes is not.

diff --git a/src/utils.rs b/src/utils.rs
index 9c58490..d9cbee8 100644
--- a/src/utils.rs
+++ b/src/utils.rs
@@ -850,6 +850,8 @@ fn test_text_width() {
         measure_text_width(&s),
         if cfg!(feature = "ansi-parsing") {
             3
+        } else if cfg!(feature = "unicode-width") {
+            17
         } else {
             21
         }
